test/storage.test.js

```diff
@@ -42,11 +42,21 @@ function _runReverseMigration(toVersion, storage) {
   }
 }
 
+// The type of the item in storage differs between the MockBrowser and Evergreen
+function storageValue(value, storageType, isMocked) {
```

Comment:
this is a bit odd - can you clarify a bit why it's different between MockBrowser and Evergreen? The "mock" should behave like the real thing as much as possible so it smells like there might be an underlying issue somewhere that we should address, if we can